### PR TITLE
COG Explorer | Major update to v0.2.0

### PR DESCRIPTION
**Pull request by ander481**
I'm proud to announce an update of COG Explorer to v0.2.0 !

Major updates:
- Application is build on Next.JS ^14.2.5
- Update to @gisatcz/ptr-maps: "^1.24.8-dev.17",
- New UI (including Mantine v7.13.4)
- Advanced CogBitMap options parsing
- MapView logic revised
- More strict TS types introduced
- Refactored all the application from scratch
- Code readability and sustainability practices were applied

Tested on: https://docs.google.com/spreadsheets/d/1utpZtHImS7sF0nnmod8fcf4BBk4LFhztgeiWcVsTXz8/edit?pli=1&gid=0#gid=0

### CODE REVIEW

File: src/components/ui/button/button.tsx
Comment (vlach1989) [inline src/components/ui/button/button.tsx:1]: Unused component?

File: src/components/ui/link/link.tsx
Comment (vlach1989) [inline src/components/ui/link/link.tsx:1]: Unused component?

File: src/components/ui/spinner/spinner.tsx
Comment (vlach1989) [inline src/components/ui/spinner/spinner.tsx:1]: Unused component?

File: src/components/ui/switch/presentation.tsx
Comment (vlach1989) [inline src/components/ui/switch/presentation.tsx:1]: Unused component?

File: src/styles/Dashboard.module.css
Comment (vlach1989) [inline src/styles/Dashboard.module.css:1]: Move styles to components

**Review by vlach1989 — CHANGES_REQUESTED** [review]

File: src/components/ui/link/link.tsx
Comment (ander481) [reply src/components/ui/link/link.tsx:1]: Yes. Resolved.

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**